app-admin/terraform/terraform-0.11.11.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="A tool for building, changing, and combining infrastructure safely"
+HOMEPAGE="https://www.terraform.io/"
+
+EGO_PN="github.com/hashicorp/${PN}"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="pie"
+DEPENDENCIES=""
+
+src_configure() {
+	EGO_BUILD_FLAGS="-buildmode=$(usex pie pie default) "
+}
+
+src_compile() {
+	pushd src/${EGO_PN} || die
+	GOPATH="${S}" GOCACHE="${T}/go-cache" go build \
+		-v -ldflags="-s -w -X main.GitCommit=${PV}-gentoo" \
+		-x ${EGO_BUILD_FLAGS} \
+		-work -o ${S}/${PN} ./ || die
+	popd
+}
+
+src_install() {
+	dobin $PN
```

Comment:
${PN} ?????

Reply:
it is not

Reply:
or rather add terraform instead of PN please its better practice

Reply:
this has been changed.